ci/windows/test_py2_cpu.ps1

```diff
@@ -16,7 +16,8 @@
 # under the License.
 
 7z x -y windows_package.7z
-$env:PYTHONPATH=join-path $pwd.Path windows_package\python
+for /r %%i in (*.dll) do xcopy /Y /C "%%i" .\\python\\mxnet\\
```

Comment:
Can we explain this? I don't get it, aren't the packed files in the windows_package folder?

Reply:
I am trying to make PYTHONPATH the git cloned one instead of stashed build, because otherwise when unittest is launched in git cloned folder, the pickled functions(from multiprocess) cannot find the correct dlls.

Reply:
Can you instead set the MXNET_LIBRARY_PATH  environment variable? this should be the correct way.